docs/source/installation.rst

```diff
@@ -95,7 +93,6 @@ Windows
 3. Install GPkit
 ++++++++++++++++
   - Run ``pip install gpkit`` at an Anaconda Command Prompt.
-  - Run ``pip install pint`` for units support (recommended)

```

Comment:
@galbramc, we're changing to listing pint as a dependency, so the CI machines should be updated accordingly.